paddle/contrib/inference/paddle_inference_api.map

```diff
@@ -0,0 +1,6 @@
+{
+	global:
+		*paddle*;
```

Comment:
@Superjomn 

Use *paddle* or the explicit name we want to expose? Which one do we prefer?

Reply:
`*paddle*` should be enough currently, we can change to the explicit names if the users need that.

Reply:
Cool